ibi/coreos-installer/README.md

```diff
@@ -15,6 +15,8 @@ This PoC will:
 1. Define a few environment variables:
 ```
 export SEED_IMAGE=quay.io/whatever/ostbackup:seed
+export SEED_VERSION=version_matching_seed_version
+export LCA_IMAGE=quay.io/customized/lca_image:your_tag
```

Comment:
LCA_IMAGE is not mandatory, since it has a default value in Makefile, I would leave it out of the readme, or state that it is optional